pkg/authorization/registry/clusterpolicy/strategy.go

```diff
@@ -65,21 +60,3 @@ func (strategy) Validate(ctx apirequest.Context, obj runtime.Object) field.Error
 func (strategy) ValidateUpdate(ctx apirequest.Context, obj, old runtime.Object) field.ErrorList {
 	return validation.ValidateClusterPolicyUpdate(obj.(*authorizationapi.ClusterPolicy), old.(*authorizationapi.ClusterPolicy))
 }
-
-// GetAttrs returns labels and fields of a given object for filtering purposes
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
-	policy, ok := obj.(*authorizationapi.ClusterPolicy)
-	if !ok {
-		return nil, nil, false, fmt.Errorf("not a ClusterPolicy")
-	}
-	return labels.Set(policy.ObjectMeta.Labels), authorizationapi.ClusterPolicyToSelectableFields(policy), policy.Initializers != nil, nil
```

Comment:
> Isn't this a behavior change? authorizationapi.ClusterPolicyToSelectableFields is not the same as your generic method?

Maybe I've been staring at this too long.  Don't both map "metadata.name" to the value of Name?